### DISCUSSION TITLE
Porting Vulkan renderer on Citra MMJ

### DISCUSSION THREAD

**Discussion by kiggggdss** [weihuoya/citra#832] [Ideas]
Since there's a Vulkan renderer now on the official nightly build of Citra, is it possible to port it to Citra MMJ?

**Comment by Nyaksss** [comment]
im patiently waiting for what weihuoya is up to.

**Comment by ghost** [comment]
> Since there's a Vulkan renderer now on the official nightly build of Citra, is it possible to port it to Citra MMJ?

It would be amazing

**Comment by unHAPPYlemon** [comment]
Either weihuoya do it or dolphin team work on the android version again. Still can't play pokemon X & Y smoothly on my phone.

**Reply by AndresNorm** [reply]
No, it is mostly based on hacks of OpenGL, Vulkan is too complex.

**Comment by LuisPerss** [comment]
Citra mmj trabaja  o trabajara en la implementacion de vulkan?.
Esto ayudaria con la compatibilidad con diversos dispositivos y procesadores.
